site/en/blog/extension-manifest-converter/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Extension Manifest Converter
+description: >
+  Open source tool to convert extensions to Manifest V3. You'll still need to manually update any
+  service worker code that relies on a DOM, in addition to updating chrome.scripts.
+subhead: >
+  Easily convert an entire directory, extension zip file, or manifest.json file.
+date: 2021-04-28
+updated: 2021-04-28
+authors:
+  - solomonkinard
+  - dotproto
+tags:
+  - extensions
+hero: image/WlD8wC6g8khYWPJUsQceQkhXSlv1/GVDuhpIli5DeYmbiPlqG.jpg
+alt: A woodworking hand plane.
+---
+
+Hi everyone. My name is Solomon and I'm a software engineer on Chrome's extensions team.
+
+As we continue to build out the latest version of [Chrome's extensions platform][mv3], I found
+myself needing to convert MV2 extensions to MV3 for testing purposes. To make this process a little
```

Comment:
MV2 + MV3 seem largely like they should be internal jargon, though they do seem to have leaked a bit.  I think we generally still use "Manifest V3" though.  Andy?

Reply:
That's on me. I get lazy and stop typing out "Manifest V3." I'm open to updating this post & avoiding the MVX pattern moving forward.